10.0/DotNetOnWebWorker/dotnet/QRGenerator.csproj

```
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk.WebAssembly">
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
```

I'm not sure about the sample placement. If we put it in net8, then no one would notice it, right? Placing it in net10, targeting net8 is not a good approach either. The upgrade should be painless, I was running it with 10 preview 6 without any changes. At the same time, it does work with earlier dotnet versions and we don't want to give an impression it's a concept for net10+.
Is there some general logic on where normally samples land? If there is no rule like: new features land in version where they were introduced, then placing it here and upgrading to target net10 is enough.
```suggestion
    <TargetFramework>net10.0</TargetFramework>
```

---

Oh ... this was my bad. I assumed incorrectly that you made it for .NET 10.

We have two choices here ...

* Make the sample .NET 10, as you suggest with the line change, and leave it here. We tell devs in the article that it's available for .NET 10 or later, which it already does.
* Make this sample .NET 10 ... we also make .NET 9 and .NET 8 samples and place them in their respective folders here. We tell devs in the article that the sample is available for .NET 8 or later ... a change that I need to make on the PR.

Which way would you like to go?

---

Also, I'll make the pluralization change after I hear back. I'll change the sample name and README to "Web Workers" language.

---

Very well ... then I think we're done here as long as you are ok leaving the demo link in the README.

---

README can have it, it's in my repo after all. Let's just make sure we do not put it in any msft docs.